src/System.Private.ServiceModel/tests/Common/Scenarios/Endpoints.cs

```diff
@@ -145,6 +145,14 @@ public static string Tcp_VerifyDNS_Address
         }
     }
 
+    public static string Tcp_VerifyDNS_HostName
+    {
+        get
+        {
+            return BridgeClient.GetResourceHostName("WcfService.TestResources.TcpVerifyDNSResource");
+        }
+    }
+
```

Comment:
For this particular endpoint, it's designed for only TCP Dns identity tests. I'm open to better naming.